.eslintrc.yml

```diff
@@ -4,5 +4,6 @@ env:
 parserOptions:
   project: tsconfig.base.json
 rules:
-  camelcase: off
-  '@typescript-eslint/no-dynamic-delete': off
+  camelcase: 0
+  '@typescript-eslint/no-dynamic-delete': 0
+  '@typescript-eslint/no-explicit-any': 0
```

Comment:
```suggestion
  camelcase: off
  '@typescript-eslint/no-dynamic-delete': off
  '@typescript-eslint/no-explicit-any': off
```

I prefer explicitness of the words

Reply:
I was working around this:
![image](https://user-images.githubusercontent.com/4118690/131297439-a46ffdda-f3a9-4884-bd0c-04c8c6c67e47.png)

Putting "off" in quotes also removes the error... I don't know which extension if any is causing this error. Seems like a false positive.

Reply:
yeah, it's weird. `off` is a string